File: haskell/snippets/fizzbuzz.hs
fizzbuzz x
  | x `mod` 3 == 0 && x `mod` 5 == 0 = "fizzbuzz"
  | x `mod` 3 == 0 = "fizz"
  | x `mod` 5 == 0 = "buzz"
  | otherwise = show x

File: ocaml/snippets/fizzbuzz.ml
let fizzbuzz x =
  match x with
  | x when x mod 3 = 0 && x mod 5 = 0 -> "fizzbuzz"
  | x when x mod 3 = 0 -> "fizz"
  | x when x mod 5 = 0 -> "buzz"
  | _ -> string_of_int x
